File: test_boilerplate.diet.ts

/* [imports omitted] */;
/* [imports omitted] */;
/* [imports omitted] */;
const logger = new Logger({
    level: LogLevel.INFO,
    format: "%(asctime)s [%(levelname)s] %(message)s"
});
<ext_safe>
// DIETTOKEN SAFE BLOCK
// We protect essential business logic using <ext_safe> tags
// DietToken will completely ignore this block during compression.
const API_KEY = process.env.API_KEY || "default_secure_key";
</ext_safe>
export interface UserModel {
    id: number;
    username: string;
    email: string;
}
export function calculateEnterpriseRevenueAndTax(baseRevenue: number, taxRate: number): number {
    const total = baseRevenue;
    const taxAmount = total * taxRate;
    const finalAmount = total + taxAmount;
    return finalAmount;
}
if (require.main === module) {
    logger.info("Initializing enterprise application...");
    console.log(calculateEnterpriseRevenueAndTax(100000.0, 0.20));
}

File: test_boilerplate.diet.py

/* [imports omitted] */
/* [imports omitted] */
/* [imports omitted] */
/* [imports omitted] */
/* [imports omitted] */
/* [imports omitted] */
/* [imports omitted] */
/* [imports omitted] */
/* [imports omitted] */
logging.basicConfig(level=logging.INFO, format="%(asctime)s [%(levelname)s] %(message)s")
logger = logging.getLogger(__name__)
<ext_safe>
# DIETTOKEN SAFE BLOCK
# We protect essential business logic using <ext_safe> tags
# DietToken will completely ignore this block during compression.
API_KEY = os.getenv("API_KEY", "default_secure_key")
</ext_safe>
class UserModel(BaseModel):
    id: int
    username: str
    email: str
def calculate_enterprise_revenue_and_tax(base_revenue: float, tax_rate: float) -> float:
    total = base_revenue
    tax_amount = total * tax_rate
    final_amount = total + tax_amount
    return final_amount
if __name__ == "__main__":
    logger.info("Initializing enterprise application...")
    print(calculate_enterprise_revenue_and_tax(100000.0, 0.20))
